src/config/server.rs

```diff
@@ -23,6 +23,8 @@ pub struct Server {
   pub proxy: Option<Proxy>,
   #[serde(default, skip_serializing_if = "is_default")]
   pub vars: KeyValues,
+  pub add_response_headers: Option<Vec<BTreeMap<String, String>>>,
```

Comment:
```suggestion
  pub add_response_headers: KeyValues,
```
Not sure if we discussed this bit before?

Reply:
Changed it to keyValues